src/main/java/org/mule/extension/sftp/internal/connection/provider/SftpConnectionProvider.java

```diff
@@ -168,6 +173,8 @@ public SftpFileSystemConnection connect() throws ConnectionException {
           LOGGER.error(e.getMessage());
           // throw new MuleRuntimeException(e);
         }
+      } else if (e.getDisconnectCode() == 3) {
```

Comment:
3 is a magic number, please replace for a constant that explains what it means

Reply:
Done, used constant from SshConstants in Apache Mina